packages/deployment/app/cloudformation.template.yml

```diff
@@ -634,6 +657,13 @@ Resources:
     Type: AWS::AutoScaling::LaunchConfiguration
     Metadata:
       AWS::CloudFormation::Init:
+        configSets:
+          MountConfig:
```

Comment:
What is the implication of this if there is no efs volume?

Reply:
It only does the first command: config, which does 01_add_instance_to_cluster

Reply:
👍